artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/ActiveMQClientLogger.java

```diff
@@ -347,4 +347,7 @@ public interface ActiveMQClientLogger {
 
    @LogMessage(id = 214034, value = "{} has negative counts {}\n{}", level = LogMessage.Level.ERROR)
    void negativeRefCount(String message, String count, String debugString);
+
+   @LogMessage(id = 214035, value = "Couldn't finish the client globalFlowControlThreadPool in less than 10 seconds, interrupting it now", level = LogMessage.Level.WARN)
```

Comment:
thanks for the added warning.
Would recommend "Could not" instead of "Couldn't" like in other messages

Reply:
My change follows the established pattern of using "Couldn't finish the..." in the other messages related to stopping the other global thread pools. I would be fine with changing this to use `Could not`, but it would need to be in another commit that changes the pattern for every use here.